### PR TITLE
Fix missing references and packages

### PR DESCRIPTION
**Pull request by paurail**

### CODE REVIEW

**Comment by tiesmaster** [comment]
Build fails, but don't worry, this is usually AppVeyor which is having problems with nuget packages. I've enabled the [build cache](http://www.appveyor.com/docs/build-cache), which should fix most of these problems.

File: Coolkit.Converters/packages.config [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="FluentAssertions" version="4.1.0" targetFramework="net452" />
```
Comment (tiesmaster) [inline Coolkit.Converters/packages.config]: Probably you added it to the wrong project, isn't it? Should be test project, right?

**Comment by tiesmaster** [comment]
This fixes the build, however, it also adds FluentAssertions to the wrong project. I'm going to close the PR, cherrypick the first commit. I hope you don't mind.

**Comment by tiesmaster** [comment]
Can you also fix your git setup, when you're committing the commit looks like [this](https://github.com/paurail/Coolkit.Converters/commit/e3da60b45c090bffd3d5da47223917f2a29b69b8.patch):

    Author: paulius.raila@live.com <paulius.raila@live.com>

Which is a weird name.

**Comment by tiesmaster** [comment]
I've also added you as collaborator on AppVeyor, based on that emailaddress. I don't know if that's the same address that you've used for GitHub, otherwise, if might not work. Please if you have access now.

**Comment by paurail** [comment]
Indeed, wrong project :)

**Pull request closed without merging**